kodilla-collections-advanced/src/main/java/com/kodilla/collections/adv/exercises/homework/FlightFinder.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.collections.adv.exercises.homework;
+
+import java.util.*;
+
+public class FlightFinder {
+
+    private Set<Flight> flights = new LinkedHashSet<>();
+    private List<Flight> result = new LinkedList<>();
+
+    public List<Flight> findFlightsFrom(String departure) {
+
+        flights = FlightRepository.getFlightsTable();
+        for (Flight flightTo : flights) {
+            if (flightTo.getDeparture().equals(departure)) result.add(flightTo);
+        }
+        return result;
+    }
+
+    public List<Flight> findFlightsTo(String arrival) {
+
+        flights = FlightRepository.getFlightsTable();
+        for (Flight flightFrom : flights) {
```

Comment:
Tu musisz przejrzrc wszystkie loty, jakby byla mapa, to by bylo tylko jedno sprawdzenie. Przeciez tyle razy o tym opowiadalem

Reply:
Tak, mapę zastosowałem w pierwszej wersji, ale zafiksowałem się że musi być flightID jako klucz i nie wiedziałem wtedy jak szukać FROM i TO.
Listy i zbiory to umożliwiły, dlatego tak zostawiłem, nie myśląc o wydajności.
W poprawce zrobiłem 2 mapy z kluczami nazw lotnisk deperture i arrival.